src/script/interpreter.cpp

```diff
@@ -1099,6 +1103,22 @@ uint256 GetJoinSplitsHash(const CTransaction& txTo) {
     return ss.GetHash();
 }
 
+uint256 GetShieldedSpendsHash(const CTransaction& txTo) {
+    CBLAKE2bWriter ss(SER_GETHASH, 0, ZCASH_SHIELDED_SPENDS_HASH_PERSONALIZATION);
+    for (unsigned int n = 0; n < txTo.vShieldedSpend.size(); n++) {
+        ss << txTo.vShieldedSpend[n];
```

Comment:
I'm confused... this seems to cover the signature inside the shielded spend. That isn't right, is it?

Reply:
Ooh, good catch!